src/uint/inv_mod.rs

```diff
@@ -25,11 +27,72 @@ impl<const LIMBS: usize> UInt<LIMBS> {
         }
         x
     }
+
+    /// Computes the multiplicative inverse of `self` mod `modulus`. In other words `self^-1 mod modulus`. Returns `(inverse, 1...1)` if an inverse exists, otherwise `(undefined, 0...0)`. The algorithm is the same as in GMP 6.2.1's `mpn_sec_invert`.
+    pub const fn ct_inv_mod(self, modulus: &UInt<LIMBS>) -> (Self, Word) {
```

Comment:
This crate has a "constant time by default unless explicitly marked otherwise" policy, so the `ct_*` prefixes on the functions are redundant.

We explicitly mark variable-time functions as `*_vartime`.

I think there were a handful of `pub(crate)` functions with `ct_*` names before which should probably be renamed, but this PR is greatly expanding that and exposing them as part of the public API.

```suggestion
    pub const fn inv_mod(self, modulus: &UInt<LIMBS>) -> (Self, Word) {
```

Reply:
Oh, scrolling down I now see there's a separate definition of `inv_mod`.

Not sure how better to name the const versions other than `const_*`, but having one named `ct_*` and the other not would generally imply to me one is constant time and the other is not, which is not the case here.

Reply:
My mistake, this function should have been `pub(crate)`. Note that this function is in fact (supposed to be) constant-time. I took the `ct_*` prefix to mean that these functions cannot work with `subtle` because they are `const`, but I see that this is indeed not always consistent. Perhaps this naming issue could be addressed in a future PR?